django_project/ford3/models_logic/qualification_audit.py

```diff
@@ -5,13 +5,10 @@ def __init__(self, qualififcation):
         self.qualification = qualififcation
 
     def evaluate_audit(self):
-        results = [eval(f'self.{method}()')
-                   for method in dir(self)
-                   if self.is_audit_method(method)]
-        if False in results:
-            return False
-        else:
-            return True
+        return False if False in [
+            eval(f'self.{method}()')
+            for method in dir(self)
+            if self.is_audit_method(method)] else True
```

Comment:
what does it return if the audit is OK?

Reply:
I updated the syntax to try and make it more clear that it returns: else True